### ISSUE TITLE
Adding option "fix" also requires you to specify "configPath"

### ISSUE THREAD

**Issue by simonauner** [jscs-dev/gulp-jscs#83]
I love the fact that the plugin now supports `fix` - great job!

However, when I added the `fix` option gulp-jscs no longer uses my own `.jscsrc` file.

I would expect the following to still default to using `./.jscsrc`.
```
return gulp.src('app.js')
        .pipe(plugins.jscs({
            fix: true
        }))
        .pipe(gulp.dest('./src'));
```
 But it doesn't. I have to specify 
```
return gulp.src('app.js')
        .pipe(plugins.jscs({
            configPath: '.jscsrc',
            fix: true
        }))
        .pipe(gulp.dest('./src'));
```

Likewise I expect
```
return gulp.src('app.js')
        .pipe(plugins.jscs({}))
        .pipe(gulp.dest('./src'));
```

to equal

```
return gulp.src('app.js')
        .pipe(plugins.jscs())
        .pipe(gulp.dest('./src'));
```

**Comment by simonauner** [comment]
I think I know how to solve it and be backwards-compatible and I'll file a PR as soon as I get time to, if anyone gets around to before me I think I have a solution below.

Culprit:
```
options = options || '.jscsrc';

if (typeof options === 'string') {
    options = {configPath: options};
}
```
Replacing it with 
```
options = options || {};

if (!options.configPath) {
    options.configPath = '.jscsrc'
}
```

**Comment by sindresorhus** [comment]
[Fixed in 3.0.0](https://github.com/jscs-dev/gulp-jscs/releases/tag/v3.0.0)
